MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -1001,6 +1197,55 @@ void EnggDiffractionViewQtGUI::plotRepChanged(int /*idx*/) {
   m_currentType = plotType->currentIndex();
 }
 
+void EnggDiffractionViewQtGUI::fittingBankIdChanged(int /*idx*/) {
+  QComboBox *BankName = m_uiTabFitting.comboBox_bank;
+  if (!BankName)
+    return;
+  m_bank_Id = BankName->currentIndex();
+}
+
+void EnggDiffractionViewQtGUI::setBankIdComboBox(int idx) {
+  QComboBox *bankName = m_uiTabFitting.comboBox_bank;
+  bankName->setCurrentIndex(idx);
+}
+
+std::string EnggDiffractionViewQtGUI::fittingRunNo() const {
+  return m_uiTabFitting.lineEdit_pushButton_run_num->text().toStdString();
+}
+
+std::string EnggDiffractionViewQtGUI::fittingPeaksData() const {
```

Comment:
I'd add a comment here saying that this should be moved into a helper. Or maybe this could use the poco string tokenizers.